resources/utils/extract.go

```diff
@@ -66,9 +66,21 @@ func extractTar(r io.Reader, targetDir string) error {
 			}
 			defer outFile.Close()
 
-			if _, err := io.Copy(outFile, tarReader); err != nil {
+			var totalWritten int64
+			if err != nil {
 				return err
 			}
+			for totalWritten != header.Size {
+				if written, err := io.CopyN(outFile, tarReader, 1024); err != nil {
+					if err == io.EOF {
+						return nil
```

Comment:
Should be "continue", otherwise we will only extract the first file in the package.